modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,166 @@
+      'description' => t('View the review tab on policy node pages'),
+    ),
+  );
+}
+
+/**
+ * The iNope Policy Review Form.
+ */
+function inope_policy_review_form($form, &$form_state) {
+
+  $status = array(
+    0 => t('I have reviewed this policy and approved it for use'),
+    1 => t('I have reviewed this policy and would like to see changes'),
+  );
+
+  $form['inope_policy_review_status_radio'] = array(
+    '#type' => 'radios',
+    '#options' => $status,
+  );
+
+  $form['inope_policy_review_changes_requested_textarea'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Changes Requested:'),
+    '#default_value' => '',
+    '#states' => array(
+      'invisible' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 0),
+      ),
+      'visible' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 1),
+      ),
+      'required' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 1),
+      ),
+    ),
+  );
+
+  $form['inope_policy_review_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Submit'),
+  );
+  
+  $form['inope_policy_review_hidden_nid'] = array(
+    '#type' => 'hidden',
+    '#value' => $form_state['build_info']['args'][0],
+  );
+  
+  $form['inope_policy_review_hidden_vid'] = array(
+    '#type' => 'hidden',
+    '#value' => node_load($form_state['build_info']['args'][0])->vid,
+  );
+
+  $form['inope_policy_review_hidden_uid'] = array(
+    '#type' => 'hidden',
+    '#value' => $GLOBALS['user']->uid,
+  );
+
+  return $form;
+}
+
+/**
+ * The iNope Policy Review Form validate handler.
+ */
+function inope_policy_review_form_validate($form, &$form_state) {
+  if ($form_state['values']['inope_policy_review_status_radio'] != 0 &&
+      $form_state['values']['inope_policy_review_status_radio'] != 1) {
+    form_set_error('inope_policy_review_status_radio', 'Please try again, status radio');
+  }
+  if ($form_state['values']['inope_policy_review_changes_requested_textarea'] !=
+      check_plain($form_state['values']['inope_policy_review_changes_requested_textarea'])) {
+    form_set_error('inope_policy_review_changes_requested_textarea', 'Please try again, only plain text in textarea');
+  }
+  if ($form_state['values']['inope_policy_review_hidden_uid'] != $GLOBALS['user']->uid) {
+    form_set_error('inope_policy_review_hidden_uid', 'Please try again, uid');
+  }
+  if ($form_state['values']['inope_policy_review_hidden_nid'] != arg(1)) {
+    form_set_error('inope_policy_review_hidden_nid', 'Please try again, nid');
+  }
+  if ($form_state['values']['inope_policy_review_hidden_vid'] != node_load($form_state['build_info']['args'][0])->vid) {
+    form_set_error('inope_policy_review_hidden_nid', 'Please try again, vid');
+  }
+}
+
+/**
+ * The iNope Policy Review Form submit handler.
+ */
+function inope_policy_review_form_submit($form, &$form_state) {
+  $account = $GLOBALS['user'];
```

Comment:
And actually it might be best if this is coming from a user load from the uid from the form.  Of course you'll need to transfer the appropriate user account load there.